### PR TITLE
Notify with promises

### PR DESCRIPTION
**Pull request by RaananW**

### CODE REVIEW

**Comment by deltakosh** [comment]
nuwatz :)

File: src/Tools/babylon.observable.ts [hunk 1]
```diff
@@ -203,6 +203,59 @@
...
+            state.skipNextObservers = false;
+
+            // execute one callback after another (not using Promise.all, the order is important)
+            this._observers.forEach(obs => {
+                if (state.skipNextObservers) {
+                    return;
+                }
+                if (obs.mask & mask) {
+                    if (obs.scope) {
+                        // TODO - I can add the variable from the last function here. Requires changing callback sig
```
Comment (deltakosh) [inline src/Tools/babylon.observable.ts]: let's do it then, no?

Comment (RaananW) [reply src/Tools/babylon.observable.ts]: ah, yes :)
Gimme like 2.5 minutes.

Comment (deltakosh) [reply src/Tools/babylon.observable.ts]: I like the german precision :)

**Pull request merged**